src/routes/tracking/trackingRouter.ts

```diff
@@ -0,0 +1,27 @@
+import { Router, Request, Response, NextFunction } from "express";
+import { LogEntity } from "../../types/entities/LogEntity";
+import { logRepository } from "../../repositories/logRepository";
+import { SHA1 } from "crypto-js";
+
+export const trackingRouter = Router();
+
+trackingRouter.post("/", async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        if(!req.body.event) 
+            throw new Error("400:Body parameter missing:event");
+            
+        if(!req.body.info) 
+            throw new Error("400:Body parameter missing:info");
+
+        const log = <LogEntity>{}
+        // hash the ip address with SHA1        
+        log.ip = SHA1(req.socket.remoteAddress ?? "").toString();
```

Comment:
use native nodejs implementation:
https://stackoverflow.com/questions/6984139/how-can-i-get-the-sha1-hash-of-a-string-in-node-js